diff -urN midori-0.0.17/configure.in midori-0.0.17.new/configure.in
--- midori-0.0.17/configure.in	2008-01-24 03:47:18.000000000 +0100
+++ midori-0.0.17.new/configure.in	2008-03-14 21:31:11.746945932 +0100
@@ -53,7 +53,7 @@
 AC_DEFINE_UNQUOTED([GTK_VER], "$GTK_VER", [GTK+ version])
 
 # Checks for WebKitGtk
-PKG_CHECK_MODULES(WEBKIT, WebKitGtk, have_webkit=true, have_webkit=false)
+PKG_CHECK_MODULES(WEBKIT, webkit-1.0, have_webkit=true, have_webkit=false)
 if test "x${have_webkit}" = "xfalse" ; then
     AC_MSG_ERROR([No WebKitGtk package information found])
 fi
diff -urN midori-0.0.17/src/global.h midori-0.0.17.new/src/global.h
--- midori-0.0.17/src/global.h	2008-01-23 03:13:40.000000000 +0100
+++ midori-0.0.17.new/src/global.h	2008-03-14 21:27:49.078864219 +0100
@@ -16,7 +16,7 @@
 #include "../katze/katze.h"
 
 #include <gtk/gtk.h>
-#include <webkit.h>
+#include <webkit/webkit.h>
 
 // -- globals
 
diff -urN midori-0.0.17/src/helpers.c midori-0.0.17.new/src/helpers.c
--- midori-0.0.17/src/helpers.c	2008-01-20 17:46:16.000000000 +0100
+++ midori-0.0.17.new/src/helpers.c	2008-03-14 21:29:01.944748071 +0100
@@ -16,7 +16,7 @@
 #include "../katze/katze.h"
 
 #include <string.h>
-#include <webkit.h>
+#include <webkit/webkit.h>
 
 GtkIconTheme* get_icon_theme(GtkWidget* widget)
 {
diff -urN midori-0.0.17/src/webSearch.h midori-0.0.17.new/src/webSearch.h
--- midori-0.0.17/src/webSearch.h	2007-12-15 04:09:47.000000000 +0100
+++ midori-0.0.17.new/src/webSearch.h	2008-03-14 21:28:42.635693788 +0100
@@ -16,7 +16,7 @@
 
 #include <gtk/gtk.h>
 #include <libsexy/sexy.h>
-#include <webkit.h>
+#include <webkit/webkit.h>
 
 // -- Types
 
diff -urN midori-0.0.17/src/webView.h midori-0.0.17.new/src/webView.h
--- midori-0.0.17/src/webView.h	2008-01-04 08:46:44.000000000 +0100
+++ midori-0.0.17.new/src/webView.h	2008-03-14 21:29:25.050223186 +0100
@@ -16,7 +16,7 @@
 #include "browser.h"
 #include "debug.h"
 
-#include <webkit.h>
+#include <webkit/webkit.h>
 
 WebKitNavigationResponse
 on_webView_navigation_requested(GtkWidget* webView, WebKitWebFrame* frame
