--- midori-0.1.0/midori/Makefile.am.orig	2008-10-23 13:03:35.000000000 +0200
+++ midori-0.1.0/midori/Makefile.am	2008-10-23 13:04:27.000000000 +0200
@@ -6,6 +6,7 @@ INCLUDES = \
     $(GTK_CFLAGS)           \
     $(GTKSOURCEVIEW_CFLAGS) \
     $(WEBKIT_CFLAGS)        \
+    $(LIBXML_CFLAGS)        \
     -I../katze
 
 AM_CFLAGS = -DMIDORI_LOCALEDIR=\""$(localedir)"\"
@@ -17,6 +18,7 @@ LDADD = \
     $(GTKSOURCEVIEW_LIBS)\
     $(WEBKIT_LIBS)       \
     $(INTLLIBS)          \
+    $(LIBXML_LIBS)       \
     ../katze/libkatze.la
 
 bin_PROGRAMS = \
