--- midori-0.1.0/midori/midori-view.c.orig	2008-10-23 13:14:55.000000000 +0200
+++ midori-0.1.0/midori/midori-view.c	2008-10-23 13:15:37.000000000 +0200
@@ -677,7 +677,7 @@ webkit_web_frame_load_done_cb (WebKitWeb
         data = g_strdup_printf (
             "<html><head><title>Not found - %s</title></head>"
             "<body><h1>Not found - %s</h1>"
-            "<img src=\"file://" DATADIR "/midori/logo-shade.png\" "
+            "<img src=\"file:///usr/share/midori/logo-shade.png\" "
             "style=\"position: absolute; right: 15px; bottom: 15px;\">"
             "<p />The page you were opening doesn't exist."
             "<p />Try to <a href=\"%s\">load the page again</a>, "
@@ -1408,7 +1408,7 @@ midori_view_set_uri (MidoriView*  view,
                 data = g_strdup_printf (
                     "<html><head><title>No documentation installed</title></head>"
                     "<body><h1>No documentation installed</h1>"
-                    "<img src=\"file://" DATADIR "/midori/logo-shade.png\" "
+                    "<img src=\"file:///usr/share/midori/logo-shade.png\" "
                     "style=\"position: absolute; right: 15px; bottom: 15px;\">"
                     "<p />There is no documentation installed at %s."
                     "You may want to ask your distribution or "
--- midori-0.1.0/midori/midori-browser.c.orig	2008-10-23 13:10:26.000000000 +0200
+++ midori-0.1.0/midori/midori-browser.c	2008-10-23 13:16:42.000000000 +0200
@@ -2549,9 +2549,9 @@ _action_help_link_activate (GtkAction*  
         #ifdef DOCDIR
         uri = DOCDIR "/midori/user/midori.html";
         if (!g_file_test (uri, G_FILE_TEST_EXISTS))
-            uri = "error:nodocs " DOCDIR "/midori/user/midori.html";
+            uri = "error:nodocs /usr/share/doc/midori/user/midori.html";
         #else
-        uri = "error:nodocs " DATADIR "/doc/midori/user/midori.html";
+        uri = "error:nodocs /usr/share/doc/midori/user/midori.html";
         #endif
     }
     else if  (!strncmp ("HelpFAQ", action_name, 7))
