--- midori-0.2.0-orig/panels/midori-addons.c	2009-10-16 00:56:27.000000000 +0200
+++ midori-0.2.0/panels/midori-addons.c	2009-11-04 18:43:22.851359300 +0100
@@ -285,9 +285,10 @@
         GTK_DIALOG_DESTROY_WITH_PARENT,
         GTK_MESSAGE_INFO, GTK_BUTTONS_CLOSE,
         _("Copy userscripts to the folder %s and "
-        "copy userstyles to the folder %s."),
-        "~/.local/share/midori/scripts",
-        "~/.local/share/midori/styles");
+        "copy userstyles to the folder %s " 
+	"(if XDG_CONFIG_HOME is unset, ~/.local/share/midori)."),
+        "XDG_CONFIG_HOME/midori/scripts", 
+        "XDG_CONFIG_HOME/midori/styles");
     gtk_dialog_run (GTK_DIALOG (dialog));
     gtk_widget_destroy (dialog);
 }
